adapter/parser/src/test/resources/test-config.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "longName": "help",
```

Comment:
I would add tests with sample data that only provide the parameters that users are likely to provide, in particular just the `longName` or `shortname` and `value`:

```json
  {
    "longName": "input",
    "value": "input"
  },
```

Reply:
Writing these test after handling default values for missing arguments